frontend/src/admin/components/userList/userList.tsx

```diff
@@ -0,0 +1,45 @@
+import '../userList/userlist.scss'
+import React, { useState } from 'react';
+import Users from "../../users.json";
+const UserList = () => {
```

Comment:
This is a function declaration, so please make use of the function keyword.
Since this is the default export of the file, you should define it here:
```suggestion
export default function UserList() {
```

Reply:
@KvelaGorrrrnio has TV2 stopped using fat arrow function declarations? 
And is the common export default at the end of a file dropped, in favor of putting it in the function declaration?

Reply:
@olzzon Arrow function are still very much in use/need, form what i know. In the case of a functional component it is however more correct to define it as Anders has suggested, from what i have learned.

Reply:
✅

Reply:
@andr9528 in a Class react component the a => creates it's own this, and that can mess stuff up.
But i have never seen anyone saying that a function declaration is more correct that double arrow when it comes to functional components. But if I'm wrong please let me know :)

Reply:
@olzzon I don't think we have had a talk on this in the team, so this will be my own opinion.
I have a couple of reasons why I prefer using `function` over const-arrow syntax in a general TypeScript context:

1. The intent is clearer. You only need to read the first word of the line to understand that this is a function and not a variable.
2. We try to follow Clean Code standards in the team. In this we aim to have the top-level function at the top, which then makes use of subsequent functions. This is to create a more linear reading experience of the code. By using const-arrow definitions, the function is only available after it has been instantiated – meaning that we have to reverse the desired order of function declarations.
3. In a stack trace, named functions will appear by name, whereas const-arrow will appear as an anonymous function.
4. It enables the function be made recursive. I know that it is not applicable to this specific instance, but in general that is a good thing.

Of course there are pros as well for using const-arrow syntax.

For exporting functions, interfaces, variables and so forth at the declaration is, in my opinion, more useful, since then the entirety of the function is self-contained – in the same manner as marking a class method private or public.

In general I would say that we try to use newer versions of EcmaScript conventions for module interaction.